test/asset_host_test.rb

```diff
@@ -0,0 +1,33 @@
+require_relative "test_helper"
+
+describe Plek do
+  describe "retreiving the public_asset_host" do
```

Comment:
You can collapse this into the previous line `describe Plek, "retrieving.." do`

You've also typo'd "retrieving".